lib/tlog/es_json_reader.c

```diff
@@ -230,6 +232,46 @@ tlog_es_json_reader_init(struct tlog_json_reader *reader, va_list ap)
         goto error;
     }
 
+    /* Allow kerberos (negotiate) authentication */
+    rc = curl_easy_setopt(es_json_reader->curl, CURLOPT_HTTPAUTH, CURLAUTH_ANY);
+    if (rc != CURLE_OK) {
+        grc = TLOG_GRC_FROM(curl, rc);
+        goto error;
+    }
+
+    /* Read credentials from .netrc if present */
+    rc = curl_easy_setopt(es_json_reader->curl, CURLOPT_NETRC, CURL_NETRC_OPTIONAL);
+    if (rc != CURLE_OK) {
+        grc = TLOG_GRC_FROM(curl, rc);
+        goto error;
+    }
+
+    /* On >= 400 responses fail immediately, before trying to parse JSON */
+    rc = curl_easy_setopt(es_json_reader->curl, CURLOPT_FAILONERROR, 1);
+    if (rc != CURLE_OK) {
+        grc = TLOG_GRC_FROM(curl, rc);
+        goto error;
+    }
+
+    /* Take credentials from TLOG_CURL_USERPWD environment variable if present */
+    if (getenv("TLOG_CURL_USERPWD")) {
+        curl_easy_setopt(es_json_reader->curl, CURLOPT_USERPWD, getenv("TLOG_CURL_USERPWD"));
+        if (rc != CURLE_OK) {
+            grc = TLOG_GRC_FROM(curl, rc);
+            goto error;
+        }
+    }
+
+    /* Enable verbose logging on the curl client. */
+    es_json_reader->verbose = verbose;
+    if (verbose) {
```

Comment:
Can you just always assign the `verbose` value to `CURLOPT_VERBOSE`, and drop this `if`?
E.g.:
```c
    rc = curl_easy_setopt(es_json_reader->curl, CURLOPT_VERBOSE, verbose);
```

Reply:
Thank you for reviewing

Reply:
Was this resolved? I still see 
~~~
    /* Configure verbose logging on the curl client. */
    if (verbose) {
~~~